src/utils/TodoFilter.tsx

```diff
@@ -0,0 +1,25 @@
+import { useMemo } from 'react';
+import { TodoStatus } from '../types/TodoStatus';
+import { Todo } from '../types/Todo';
+
+export const FilterTodos = (todos: Todo[], todoStatus: TodoStatus) => {
```

Comment:
Why is it capitalized?

Reply:
React Hook "useMemo" is called in function "filterTodos" that is neither a React function component nor a custom React Hook function. React component names must start with an uppercase letter. React Hook names must start with the word "use".

Reply:
So what makes it a component?